File: src/main/java/com/cag/cagbackendapi/entities/AgeIncrementEntity.java
package com.cag.cagbackendapi.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.Type;

import javax.persistence.*;
import java.util.UUID;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "age_increment")
public class AgeIncrementEntity {
    @Id
    @Type(type = "pg-uuid")
    private UUID age_increment_id;
    private int youngest_age;
    private int oldest_age;
}

File: src/main/java/com/cag/cagbackendapi/entities/AgeIncrementEntity.kt
package com.cag.cagbackendapi.entities

import org.hibernate.annotations.GenericGenerator
import org.hibernate.annotations.Type
import java.util.*
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "age_increment")
data class AgeIncrementEntity(
    @Id
    @Type(type = "pg-uuid")
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(
            name = "UUID",
            strategy = "org.hibernate.id.UUIDGenerator"
    )
    var age_increment_id: UUID?,
    var name: String?
)
